### PR TITLE
Added facilitator bios and map pins

### PR DESCRIPTION
**Pull request by aoby**
Note: the added function `formatted_date_with_start_and_end_times` in session.rb is also in https://github.com/code-dot-org/code-dot-org/pull/8499, along with tests.

cc @breville

### CODE REVIEW

File: dashboard/app/views/pd/workshop_enrollment/new.html.haml [hunk 1]
```diff
@@ -3,88 +3,102 @@
...
+                  *
+              %div.controls
+                =form.text_field :email_confirmation, class: 'span10'
+          %div.control-group
+            %label
+              District
+              %span.form-required-field
+                *
+            %div.controls
+              =form.text_field :district_name, class: 'span10'
```
Comment (breville) [inline dashboard/app/views/pd/workshop_enrollment/new.html.haml]: you'll need to integrate the new district dropdown here..

File: dashboard/app/views/pd/workshop_enrollment/new.html.haml [hunk 2]
```diff
@@ -3,88 +3,102 @@
...
     Taught by Code.org facilitators who are experienced computer science educators,
     our workshops will prepare you to teach the Code.org curriculum.
 
-  %h2
-    Workshop Details
+  .container-fluid
+    .row-fluid
+      -# Left Column
+      .span6
+        .row-fluid
```
Comment (breville) [inline dashboard/app/views/pd/workshop_enrollment/new.html.haml]: what prompted the move to fluid rows?

File: pegasus/sites.v3/code.org/public/js/workshop-search.js [hunk 3]
```diff
@@ -35,7 +42,7 @@ function loadWorkshops() {
 
         var infowindow_content = '';
 
-        if(typeof markers[hash]==='undefined'){
+        if (typeof markers[hash]==='undefined') {
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/js/workshop-search.js]: need spaces around === ?

File: pegasus/sites.v3/code.org/public/js/workshop-search.js [hunk 4]
```diff
@@ -48,15 +55,61 @@ function loadWorkshops() {
...
+      if (status == google.maps.GeocoderStatus.OK) {
+        var location = results[0].geometry.location;
+        var lat = location.lat();
+        var lng = location.lng();
+        var hash = new google.maps.LatLng(lat, lng).toUrlValue();
+
+        var infowindow_content = '';
+
+        if (typeof markers[hash]==='undefined') {
+          infowindow_content = compileHtmlNew(workshop, true);
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/js/workshop-search.js]: should we camel case all these variables?  e.g. infoWindowContent?

File: dashboard/app/views/pd/workshop_enrollment/new.html.haml (hunk 2 above)
Comment (aoby) [inline dashboard/app/views/pd/workshop_enrollment/new.html.haml]: I tried a number of things to get the 2 columns to work properly. I think the outer container is what did it. I can probably remove the -fluid parts.

File: pegasus/sites.v3/code.org/public/js/workshop-search.js (hunk 3 above)
Comment (aoby) [inline pegasus/sites.v3/code.org/public/js/workshop-search.js]: I cleaned up this code the minimum needed to get it to pass the linting rules and function properly. It's not needed, but I prefer it as well so I'll add them. In general this code can be cleaned up quite a bit.

File: pegasus/sites.v3/code.org/public/js/workshop-search.js (hunk 4 above)
Comment (aoby) [inline pegasus/sites.v3/code.org/public/js/workshop-search.js]: might as well. I copied and modified this function from above. I'll take another quick pass at cleaning it up slightly.

File: dashboard/app/views/pd/workshop_enrollment/new.html.haml (hunk 2 above)
Comment (aoby) [inline dashboard/app/views/pd/workshop_enrollment/new.html.haml]: yeah, I switched it back to normal rows. Good catch.

**Comment by aoby** [comment]
I merged staging to integrate the district explorer, and updated. I also added css for .form-required-field to make the required *s red, which was somehow missing.

**Comment by tanyaparker** [comment]
LGTM

**Pull request merged**